verifier/verifier.py

```diff
@@ -43,30 +43,36 @@ def __init__(self, code, response):
         self.code = code
         self.response = response
 
+
+####
+# SMTP RCPT error handlers
+####
 def handle_550(response):
     if any([keyword.encode() in response for keyword in blocked_keywords]):
         return "Blocked by mail server"
     return "Mailbox unavailable"
 
-def handle_551(response):
-    return "Mailbox has moved"
-
-def handle_421(response):
-    return "Try again later"
-
-def handle_450(response):
-    return "Mailbox busy"
-
-def handle_other(response):
-    return "Other Error"
 
+# https://www.greenend.org.uk/rjk/tech/smtpreplies.html#RCPT
+# https://sendgrid.com/blog/smtp-server-response-codes-explained/
 handle_error = {
+    # 250 and 251 are not errors
     550: handle_550,
-    551: handle_551,
-    450: handle_450,
-    421: handle_421,
+    551: lambda _: "Mailbox has moved",
+    552: lambda _: "Storage allocation exceeded",
```

Comment:
This should result in `full_inbox` = False

Reply:
As well as in `deliverable` = False, I'm guessing?

Reply:
Now that I think about it - should all errors cause `deliverable `= False, or is there something I'm missing?

Reply:
> I assume this was a mistake and you meant `True`
You are right.

`deliverable` means that the email exists and if you try to send email it will go through.
`full_inbox` means that the mailbox is full. So when it is deliverable actually. It will be up to user to decide what to do.
`message` should be there only when we cannot give a definitive answer. `message` in return value generally means we weren't able to determine the reult and here's the reason for that.